intelmq/lib/upgrades.py

```diff
@@ -38,6 +38,7 @@
            'v301_deprecations',
            'v310_feed_changes',
            'v310_shadowserver_feednames',
+           'v320_update_turris_greylist_url'
```

Comment:
```suggestion
           'v320_update_turris_greylist_url',
```
sorry for bothering again

Reply:
No problem. :)